### PR TITLE
introduce pdf export

### PR DESCRIPTION
**Pull request by steveoh**
this is currently not printing the appliance wmts layer. I will have to investigate more whats up with that.

refs# https://redmine.dts.utah.gov/issues/36801

### CODE REVIEW

File: src/app/mapController.js [hunk 1]
```diff
@@ -132,6 +134,9 @@ define([
 
             var centroidButton = new CentroidSwitchButton({}).placeAt(toolbarNode, 'last');
 
+            var printButton = new PrintButton({
+            }).placeAt(toolbarNode, 'last');
```
Comment (stdavis) [inline src/app/mapController.js]: Any reason this is split out on a separate line. Looks funny to me.

**Comment by stdavis** [comment]
Need to handle when the job fails. My job is failing:
image
But `onError` never fired:
image

Also, how did you get this to work on your machine? Mine can't resolve localhost:8000/arcgis...

File: src/app/mapController.js (hunk 1 above)
Comment (steveoh) [inline src/app/mapController.js]: Nope

**Comment by steveoh** [comment]
On error seems to only react to the gp messages. I'm not sure there is another event to watch. My proxy is setup to watch on 8000 and forward.

**Comment by steveoh** [comment]
You also need to publish the gp async.

**Comment by stdavis** [comment]
Closed with 14040d79196736b2ab5fb0966f371e709669777c

**Pull request closed without merging**